api/users/serializers.py

```diff
@@ -252,3 +252,17 @@ def get_absolute_url(self, obj):
 
     class Meta:
         type_ = 'institutions'
+
+
+class UserIdentitiesSerializer(BaseAPISerializer):
+    data = ser.DictField()
```

Comment:
Instead of defining a `data` DictField, inherit from the JSONAPISerializer which will automatically place your information inside of a data dict.  In this serializer you should define the individual fields you want to display for each identity, which are `id`, `type`, `external_id`, and `status`.  If you are returning a list of all identities, the JSONAPISerializer takes care of returning a list of dictionaries. If you request a single identity, the serializer returns a data dictionary.